### PR TITLE
1748: Track when materials is added to lists

### PR DESCRIPTION
**Pull request by holt83**
https://platform.dandigbib.org/issues/1748

### CODE REVIEW

File: modules/ding_webtrends/js/webtrends_add.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
       // Array of selectors for links to track via webtrends.
       var webtrendsAdd = [
         '.action-button.reserve-button',
-        '.action-button.bookmark-button',
+        '.ding-list-add-button .use-ajax',
```
Comment (kasperg) [inline modules/ding_webtrends/js/webtrends_add.js]: Why is `.use-ajax` included as a selector here? Whether or not to use AJAX does not seem relevant to the tracking.

Comment (holt83) [reply modules/ding_webtrends/js/webtrends_add.js]: We should only track buttons which uses AJAX. 

The button to add a new list for example, results in page reload and gets tracked the normal way. Using .use-ajax in the selector leaves this button out.

Comment (kasperg) [reply modules/ding_webtrends/js/webtrends_add.js]: But then the element we want should have a more specific class. AJAX is a byproduct.

Comment (holt83) [reply modules/ding_webtrends/js/webtrends_add.js]: I'm not sure I understand: "AJAX is a byproduct". Can you elaborate?

We want to target the links using ajax, so I'm having a hard time seeing the problem here.

What do you suggest for a more specific class?

Comment (holt83) [reply modules/ding_webtrends/js/webtrends_add.js]: I updated the PR to add a more specific class to the links which attaches materials to lists (still checking for use-ajax as a precaution).
